integration-tests/testconfig/README.md

```diff
@@ -116,6 +116,83 @@ When processing TOML files, the system initially searches for a general (unnamed
 
 Finally `default.toml` file is envisioned to contain fundamental and universally applicable settings, such as logging configurations.
 
+### Chainlink Node TOML config
+
+To set custom config for Chainlink Node use `NodeConfig.BaseConfigTOML` in TOML. Example:
```

Comment:
I think we need richer description. That would tell at least:
* where is the default node config
* how to override it
* mention that you cannot override just a single value, e.g. `FeedsManager = true` but whole config

Reply:
@Tofel isn't default and overrides described in `Configuration and Overrides`? Or this is something else?

Reply:
@Tofel pls check 04cd351581d6d3c23f9d9959e8854f969d4e4083